media-video/ffmpeg/ffmpeg-4.4.1-r4.ebuild

```diff
@@ -563,4 +564,6 @@ multilib_src_install() {
 multilib_src_install_all() {
 	dodoc Changelog README.md CREDITS doc/*.txt doc/APIchanges
 	[ -f "RELEASE_NOTES" ] && dodoc "RELEASE_NOTES"
+
+	[ use amf ] && doenvd ${FILESDIR}/amf-env-vulkan-override
```

Comment:
```suggestion
	use amf && doenvd "${FILESDIR}"/amf-env-vulkan-override
```

Reply:
Thanks ! I would love to have a documentation that says which variables need to be quoted and which ones don't x)